src/GovernorEvents.ts

```diff
@@ -137,6 +137,11 @@ export function handleProposalCreated(event: ProposalCreatedEvent): void {
   if (!proposal.reverted) {
     proposalEntity.quorum = proposal.value.getQuorum()
     proposalEntity.majorityPercentage = proposal.value.getMajorityPercentage()
+    // set the quorum and majority percentage to 5 and 50 respectively if the governor is an admin
+    if (governor.type === GovernorType.Admin) {
+      proposalEntity.quorum = BigInt.fromI32(5)
+      proposalEntity.majorityPercentage = BigInt.fromI32(50)
+    }
```

Comment:
Issue is with the FE not subgraph, subgraph sets values correctly because they are coming in from the contract of proposal (which is chosen incorrectly in FE side)
```suggestion
```